frontend/pages/policies/PolicyPage/components/PolicyForm/PolicyForm.tsx

```diff
@@ -497,6 +499,18 @@ const PolicyForm = ({
         />
       )}
       {renderLiveQueryWarning()}
+      {isObserverPlus && ( // Observer+ can run existing policies
```

Comment:
This comment seems redundant with the one above

Reply:
Removed this part from the top comment